src/workspace/Lesson/Section/index.jsx

```diff
@@ -61,22 +61,69 @@ const Warning = styled(Text)`
 	margin-bottom: 25px;
 `
 
-const Section = ({ store, dispatch }) => {
+const getSectionData = (store, sectionId = '', setError) => {
+	const index = store.sections.findIndex(s => s._id === sectionId)
+	try {
+		return [store.sections[index], index];
+	} catch (err) {
+		if (index===-1) {
+			setError(new Error({message: 'Section existiert nicht...'}))
+		} else {
+			setError(err);
+		}
+		return [null, index];
+	}
+}
+
+const getErrorView = (error) => {
+	return (
+		<Container>
+			<Flex>
+				<Text center>{error.message || error}</Text>
+			</Flex>
+		</Container>
+	)
+}
+
+const Section = () => {
+	const { store, dispatch } = useContext(LessonContext)
 	const sectionRef = useRef(null)
 
-	if (!store.view.activeSectionId)
+	const [error, setError] = useState(null)
+	const [timeout, setTimeoutState] = useState(null)
+	const onChange = ({changed, getDocument}) => {
+		if (timeout) {
+			clearTimeout(timeout);
+			setTimeoutState(null)
+		}
+
+		setTimeoutState(setTimeout(() => {
+			setTimeoutState(null)
+			dispatch(updateSectionDocValue(store.view.activeSectionId, getDocument()))
+		}, 250))
+	}
+	const [section, index] = getSectionData(store, store.view.activeSectionId, setError)
+
+	if (!store.view.activeSectionId) {
+		setError(new Error({message: 'Kein Abschnitt ausgewählt...'}))
```

Comment:
should it also trigger sentry and mabey an error to the store?

Reply:
I think sentry at this position is not necessary. In the case that a real error is throw yes. Maybe we can change the error helper store and use dispatch for real errors and all the other stuff is "do anything then it work, or wait to load data".